### ISSUE TITLE
useSKStoreReviewControllerIfAvailable is ignored

### ISSUE THREAD

**Issue by dtdw** [nicklockwood/iRate#296]
useSKStoreReviewControllerIfAvailable is set to YES by default but not tested in code. The call should be like

if (!manual && [SKStoreReviewController class] && self. useSKStoreReviewControllerIfAvailable)
{
...
}

in - (void)promptForRating:(BOOL)manual

**Comment by nicklockwood** [comment]
Yes, this is a bug. It sounds like *not* using SKStoreReviewControllerIfAvailable may be prohibited on iOS 11 anyway, so I may just remove the option.

**Comment by mTuran** [comment]
Bug confirmed. useSKStoreReviewControllerIfAvailable is ignored.

**Comment by nicklockwood** [comment]
AFAIK, Apple is now rejecting apps that don't use SKStoreReviewControllerIfAvailable, so this seems like a low priority to fix.

**Comment by mTuran** [comment]
Yes, I heard this before but I submitted a lot of apps that does not use SKStoreReviewControllerIfAvailable and they approved without issues.
